src/pages/FormPage.tsx

```diff
@@ -12,159 +13,186 @@ const FormPage = () => {
   const handleSubmit = (event: FormEvent) => {
     event.preventDefault();
     if (form.current) {
+      const countryDropdownValue =
+        countryDropdown.current?.getAttribute("selected-value");
```

Comment:
does countryDropdown.current?.selectedValue not work? Just curious

Reply:
Good catch, let me try it out. I thought of similar once I saw the dropdown-v2 code example and passing "options"...

Reply:
Ah alright!